### PR TITLE
Add Cross Room Jumps in Pink Maridia

### PR DESCRIPTION
**Pull request by kjbranch**
- Halfie Climb Room
- Precious Room
- West Cactus Alley

### CODE REVIEW

File: region/maridia/inner-pink.json [hunk 1]
```diff
@@ -3194,6 +3259,38 @@
...
+                {
+                  "name": "Cross Room Jump with HiJump",
+                  "notable": false,
+                  "requires": [
+                    "HiJump",
+                    "canCrossRoomJumpIntoWater",
+                    "canMomentumConservingTurnaround",
+                    {"adjacentRunway": {
+                      "fromNode": 2,
+                      "usedTiles": 1,
```
Comment (osse101) [inline region/maridia/inner-pink.json:3272]: half tile TBH, just need 1 run frame

**Review by osse101 — COMMENTED** [review]
I think Rando does a wiggle to climb precious when you get ridleys boss room there.  But this momentumTurnAround is just easier.

File: region/maridia/inner-pink.json [hunk 2]
```diff
@@ -2233,6 +2233,21 @@
                     "HiJump",
                     "SpaceJump"
                   ]
+                },
+                {
+                  "name": "Cross Room Jump with Wall Jump",
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-pink.json:2238]: I put this on the wrong node, I'll fix it, but should this IBJ also require 2 Mochtroid hits or canDoubleBombJump?

[the remark this answers is not in the document]
Comment (osse101) [reply region/maridia/inner-pink.json:2238]: 2 Mochtroid hits no suit is a lot so probably both OR'd

**Pull request merged**